hooks/useStore.js

```diff
@@ -2,6 +2,10 @@ import create from 'zustand';
 import { nanoid } from 'nanoid';
 import { persist } from 'zustand/middleware';
 
+const toastTypes = {
+  ADD: 1,
+};
+
```

Comment:
Is this object going to change or does it become bigger with time?
Probably you just need it for toggling.

Reply:
For the moment it is a bit too much actually. But for better readability I would like to keep it. Otherwise I would just call id: 1 in line 28 for example.